SeriesCruncher/app/src/main/res/anim/dialog_enter_animator.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<set xmlns:android="http://schemas.android.com/apk/res/android"
+    android:interpolator="@android:anim/decelerate_interpolator">
+    <alpha
+        android:duration="250"
```

Comment:
Please consider using the default android animation durations everywhere if possible: config_shortAnimTime or config_mediumAnimTime.